client/post-editor/controller.js

```diff
@@ -235,6 +248,13 @@ module.exports = {
 		renderEditor( context, postType );
 	},
 
+	revision: function( context ) {
+		const siteId = getSelectedSiteId( context.store.getState() );
+		const postId = getPostID( context );
+
+		renderRevision( context, siteId, postId );
```

Comment:
it should :) That said, I removed that code (for now) to keep the PR focused on redux changes. Will keep it in mind for the next PR, thx :)